docs/whats-new/preview-4.md

```diff
@@ -0,0 +1,608 @@
+
+Docker or Podman will be auto-detected; if both are present, Docker is preferred. Podman can be explicitly enabled/forced via an environment variable `DOTNET_ASPIRE_CONTAINER_RUNTIME=podman`.
+
+## Dashboard
+
+### Dashboard face lift
+
+:::image type="content" source="media/preview-4/dashboard-face-lift.png" lightbox="media/preview-4/dashboard-face-lift.png" alt-text=".NET Aspire Dashboard: Updates showing landing page.":::
+
+The dashboard has been updated with a new look and feel. The new dashboard is designed to reduce the space used by the navigation tabs and to make it easier to navigate between logs, metrics, and traces.
+
+### Running the Aspire dashboard standalone
+
+The Aspire dashboard can now be run as a standalone container image. This makes it easier to use the dashboard to manage applications that are running on a different machine or in a different environment. The dashboard can be used as an [OTLP](https://opentelemetry.io/docs/specs/otlp/) collector and viewer for applications that want to send and visualize telemetry data.
+
+Here's the command you can run to start the dashboard:
+
+```bash
+docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.4
+```
+
+:::image type="content" source="media/preview-4/dashboard-as-container.png" lightbox="media/preview-4/dashboard-as-container.png" alt-text=".NET Aspire Dashboard: running in Docker Desktop.":::
+
+There are 2 ports that are exposed:
+
+1. The port that serves the dashboard UI (18888)
+2. The port that serves the OTLP grpc endpoint (18889)
+
+That will bring up a dashboard that you can use view logs, metrics, and traces from your applications. Here's a sample application that sends telemetry data to the dashboard.
+
+TODO: Sample pointing to the dashboard.  (@drewnoakes @JamesNK @kvenkatrajan)
+
+### Dashboard shortcuts
+
+The Aspire Dashboard now supports keyboard navigation via keyboard shortcuts. Click <kbd>Shift</kbd>+<kbd>?</kbd> to display the list of available shortcuts:
+
+:::image type="content" source="media/preview-4/dashboard-shortcuts.png" lightbox="media/preview-4/dashboard-shortcuts.png" alt-text=".NET Aspire Dashboard shortcuts dialog.":::
+
+### Metrics table view
+
+With preview 4, we have introduced a screen reader compatible table view for display of metrics data. This has similar options, as the graph, for filters and selecting the duration of time range for metrics display. The default data display is set to "Only show value updates" and can be toggled to display all data points.
+
+![image](https://github.com/dotnet/docs-aspire/assets/102772054/4622cf3a-dcf2-4149-9636-d1bef8184c5c)
+
+## Entity Framework and Aspire
+
+TODO: Changes to EF based components (@sebastienros @eerhardt)
```

Comment:
### New Entich methods

Preview 4 introduces new methods for configuring Entity Framework. The current `Add[Provider]DbContext()` methods used to register and configure `DbContext` classes are not sufficient for advanced cases like using a different lifetime scope, using custom service types, or configure the underlying data sources.

To solve these advanced scenarios new methods named `Enrich[Provider]DbContext()` have been added. These methods do not register the `DbContext` and expect you to do so before invoking them.

Usage example:

```csharp
var connectionString = builder.Configuration.GetConnectionString("catalogdb");
builder.Services.AddDbContextPool<CatalogDbContext>(dbContextOptionsBuilder => dbContextOptionsBuilder.UseSqlServer(connectionString));
builder.EnrichSqlServerDbContext<CatalogDbContext>();
```

These methods will still configure command retries, health checks, logging and telemetry.

### Changes to previously existing methods

Since these new methods provide a simpler way to configure the `DbContext`, the already existing ones (`Add[Provider]DbContext()`) have been simplified.

1- They don't provide a way to disable connection pooling through settings anymore. Instead register the `DbContext` with connection pooling disabled and call the corresponding `Enrich` method.
2- The `int MaxRetryCount` settings were removed and replaced by a single `bool Retry` flag that is enabled by default. To change the value configure the `DbContext` registration using the specific provider options and invoke the corresponding `Enrich` method.

Reply:
I integrated these changes, thanks @sebastienros !